modules/dedicated-accessing-your-cluster.adoc

```diff
@@ -7,8 +7,29 @@
 
 To access your {product-title} cluster:
 
-. Log in to link:https://cloud.redhat.com/[cloud.redhat.com].
+. Log in to link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift].
 
-. Select *Create Cluster* -> *Red Hat-Managed Cluster*.
+. Select *Create Cluster* -> *Red Hat OpenShift Dedicated*.
 
-. Enter your cloud provider credentials.
+. Enter your *Cluster name*, number of *Compute nodes*, and select an *AWS Region*.
+
+. Select your *Node Type*. The number and types of nodes available to you depend
+upon your {product-title} subscription.
+
+. If you wish to configure your networking IP ranges un *Advanced Options*, the
+following are the default ranges available to use:
+
+.. Node CIDR: 10.0.0.0/16
+
+.. Service CIDR: 172.30.0.0/16
+
+.. Pod CIDR: 10.128.0.0/14
+
+. Add your Identity provider by clicking the *Add OAuth Configuration* link.
+Further configuration details may be found
+link:https://docs.openshift.com/container-platform/4.1/authentication/understanding-identity-provider.html#supported-identity-providers[here].
```

Comment:
Should the dedicated docs be linking to the OCP docs? I don't see the authentication assemblies included in the dedicated docs? Some questions:

* Should this be an xref?
* If it remains an xref, then do we need to include the relevant authentication assemblies?
* If it's a link then we should you use `{product-version}` instead of hardcoding the version for 4.1?

Reply:
So in my chat with @wgordon17 today we talked about linking to OCP for that section. Can you xref to OCP or would it just xref to dedicated?  Would using `{product-version}` in the link default back to dedicated?

Reply:
Decided to build an entire dedicated-specific auth book, so the link has been changed to a dedicated xref. Thanks for your help @huffmanca !